src/encryption_driver.ts

```diff
@@ -0,0 +1,58 @@
+/*
+ * @adonisjs/encryption
+ *
+ * (c) AdonisJS
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+import { createHash } from 'node:crypto'
+import { base64 } from '@poppinss/utils'
+import * as errors from './exceptions.js'
+import { MessageVerifier } from './message_verifier.js'
+import type { BaseConfig } from './types.js'
+
+export class EncryptionDriver {
```

Comment:
What's the purpose of the `EncryptionDriver` class?

Reply:
Since we are going to have multiple drivers in the send, I wanted to regroup parts that I know will be used across other drivers too.